src/Domain/ImageReference.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Liip\ImagineBundle\Domain;
+
+interface ImageReference
```

Comment:
i renamed this to ImageReference as its the reference to an image rather than an actual image. i noticed when i got confused in LiipTransformerStackExecutor where we also have the imagine ImageInterface which holds the actual image data and has filters applied to it.